app/webpacker/controllers/tom_select_controller.js

```diff
@@ -24,4 +29,10 @@ export default class extends Controller {
   disconnect() {
     if (this.control) this.control.destroy();
   }
+
+  #placeholder() {
```

Comment:
Why using a `#` for the method name here? 
I used to know the sharp variables name, but it's deprecated: http://udn.realityripple.com/docs/Archive/Web/Sharp_variables_in_JavaScript

Reply:
☝️

Reply:
Oh I didn't know about that one. Maybe we should use it, instead of using this `// private` command. I don't what others thinks.

Reply:
I didn't know about private fields/methods either, great to see Javascript has that ability.

I think using the built-in language construct is more useful than a comment that doesn't enforce anything, so I'd suggest we do introduce that. 
But given that it's a new feature that's not used in our codebase, it would be a good idea to also have a `// private` comment, and ensure private variables are below this. IE:

```js
  // private
  
  #placeholder() {
```

If we agree, we can post this on #dev and let others know.

Reply:
Great, this is updated so I've posted on #dev.